app/assets/javascripts/models/data/collection.js.coffee

```diff
@@ -62,6 +62,18 @@ ns.Collection = do (ko
       @details = @asyncComputed({}, 100, @_computeCollectionDetails, this)
       @detailsLoaded = ko.observable(false)
       @gibs = ko.observable(null)
+      
+      @gibsLayers = ko.computed( (->
+        available = []
+        if @gibs()
+          available.push('Antarctic') if @gibs()[0].antarctic
+          available.push('Arctic') if @gibs()[0].arctic
+          available.push('Geo') if @gibs()[0].geo
```

Comment:
I prefer "Geographic". While neither Geo nor Geographic are the official names of the projection, Geographic is what was requested by end users and a customer for the map tool. So it at least carries meaning with the community.